frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/ERDContent.tsx

```diff
@@ -164,44 +177,82 @@ export const ERDContentInner: FC<Props> = ({
         (e) => e.source === id || e.target === id,
       )
 
-      const updatedEdges = edges.map((e) =>
-        relatedEdges.includes(e) ? highlightEdge(e) : e,
+      const relatedToActiveNodeEdges = edges.filter(
+        (e) => e.source === activeNodeId || e.target === activeNodeId,
       )
 
+      const updatedEdges = edges.map((e) => {
+        if (relatedEdges.includes(e)) {
+          return highlightEdge(e)
+        }
+        if (relatedToActiveNodeEdges.includes(e)) {
+          return highlightEdge(e)
+        }
```

Comment:
Is this the solution to the PR title cause?

Reply:
I suspect that ``return unhighlightEdge(e)`` is referring to the code in the next line.
It seems that the ``unhighlightEdge`` function was not working correctly when the hovered TableNode changed.